src/main/resources/org/jenkinsci/plugins/configfiles/ConfigFilesUI/selectprovider.jelly

```diff
@@ -28,6 +28,20 @@ THE SOFTWARE.
 	<l:layout title="${%filetype}" norefresh="true">
 		<cf:sitepanel />
 
+		<!-- Remove when Jenkins LTS release is no longer supported by update center-->
+		<j:if test="${h.version.startsWith('2.289.') or h.version.startsWith('2.303.') or h.version.startsWith('2.319.')}">
```

Comment:
can probably make this always here with a TODO comment?

it's on it's own standalone page

Reply:
I meant remove the if statement not change to TODO

Reply:
Ah I misunderstood - hope the latest commit is more inline with what you're thinking.